ballerina/channel_csv_io.bal

```diff
@@ -40,30 +40,31 @@ Error {
     return (check getReadableCSVChannel(readableChannel, 0)).csvStream();
 }
 
-isolated function channelWriteCsv(WritableChannel writableChannel, string[][]|map<anydata>[] content) returns Error? {
-    WritableCSVChannel csvChannel = check getWritableCSVChannel(writableChannel);
-    if content is string[][] {
-        foreach string[] r in content {
+isolated function channelWriteCsv(string path, FileWriteOption option, string[][]|map<anydata>[] contentToWrite) returns Error? {
+    WritableCSVChannel csvChannel;
```

Comment:
```suggestion
    WritableCSVChannel csvChannel;
```

Reply:
Let's do this, when we create the channel.

Reply:
updated in [464a54b](https://github.com/ballerina-platform/module-ballerina-io/pull/827/commits/464a54b55a25c3ad8a1fce969d66066be7c0a324)